README.md

```diff
@@ -24,3 +24,21 @@ If you have a readme file, and your site doesn't otherwise have an index file, t
   gems:
     - jekyll-readme-index
   ```
+
+## Configuration
+
+Configuration options are optional are placed in `_config.yml` under the `readme_index` key. They default to:
+
+```yml
+readme_index:
+  remove_originals: false
+  disabled:         false
```

Comment:
Wondering if semantically it makes more sense to make this `enabled: true` to say that it's "enabled by default"

Reply:
agreed. I'll change it across my other PRs for consistency.